### PR TITLE
refactored init as a plugin

### PR DESCRIPTION
**Pull request by tdfranklin**
fixes issue #23 

Let me know if this is what you were looking for!

### CODE REVIEW

File: plugins/init/package.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+    "name": "@reframe/init",
+    "version": "0.0.1-rc.12",
+    "publishConfig": {
+        "access": "public"
+    },
+    "repository": "github:reframejs/reframe"
+}
```
Comment (brillout) [inline plugins/init/package.json]: We should move the deps as well.  E.g.`inquirer` is now loaded by `@reframe/init`

**Comment by brillout** [comment]
Looks good otherwise:)

**Comment by tdfranklin** [comment]
Good call!  I think I have that resolved now.  Do you see anything I missed?

**Comment by brillout** [comment]
`reassert` is loaded but not declared as a dep, I'll take care of it.

Otherwise good 👍

**Pull request merged**